xml-registries/1.0/_index.md

```diff
@@ -0,0 +1,39 @@
+<!-- Template for the root page of a specification release -->
+---
+title: "Jakarta XML Registries Spec 1.0"
+date: 2019-08-12
+summary: "Initial version of the Jakarta XML Registries spec for Jakarta EE 8."
+---
+Jakarta XML Registries describes Java API’s designed specifically for an open and interoperable set of registry
+services that enable sharing of information between interested parties.
+The shared information is maintained as objects in a compliant registry.
+All access to registry content is exposed via the interfaces defined for the Registry Services.
+
+* [Jakarta XML Registries 1.0 Specification Document](./xml-registries-spec-1.0.pdf) (PDF)
+* [Jakarta XML Registries 1.0 Specification Document](./xml-registries-spec-1.0.html) (HTML)
+* [Jakarta XML Registries 1.0 Javadoc](./apidocs)
+* [Jakarta XML Registries 1.0 TCK](https://download.eclipse.org/jakartaee/xml-registries/1.0.0/xml-registries-tck-1.0.0.zip)
```

Comment:
Should this not be:

https://download.eclipse.org/jakartaee/xml-registries/1.0/xml-registries-tck-1.0.0.zip)

https://download.eclipse.org/jakartaee/{spec}/x.y/*{spec}-tck-x.y.z.zip

Reply:
Nice catch.  Based on the discussion on today's call, it should be:

https://download.eclipse.org/jakartaee/xml-registries/1.0/eclipse-xml-registries-tck-1.0.0.zip 

or, substitute jakarta-:

https://download.eclipse.org/jakartaee/xml-registries/1.0/jakarta-xml-registries-tck-1.0.0.zip

I'll get this cleaned up.